docs/getting-started/index.md

```diff
@@ -36,15 +36,17 @@ terramate {
 }
 ```
 
-Terramate uses the filesystem as its hierarchy and needs to know the project’s root. In most circumstances, this would be the git root, but we’re explicitly marking the root with a blank config for now.
+Terramate uses the filesystem as its hierarchy and needs to know the project’s root. This also means that all absolute paths that are used to reference stacks are based on the project’s root and not the root of the filesytem.
```

Comment:
```suggestion
Terramate uses the filesystem as its hierarchy and needs to know the project’s root, since all paths are relative to the project root, not the filesystem root.
```

Reply:
the suggested change drop that absolute path are having using the project directory as root.